CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+Unreleased
+----------
+
+* Removed usage of pkg_resource to avoid derecation
+  warnings. If you want to inspect the installed version,
+  use ``importlib.metadata.version("django-widget-tweaks")``
+  (`docs <https://docs.python.org/3.8/library/importlib.metadata.html#distribution-versions>`__ /
+  `backport <https://pypi.org/project/importlib-metadata/>`__).
```

Comment:
Fix some typos, and remove the backport link, since nearly everyone runs Python 3.8+ now, and 3.7 is EOL in a couple months.

```suggestion
* Removed usage of ``pkg_resources`` to avoid deprecation
  warnings. If you want to inspect the installed version,
  use ``importlib.metadata.version("django-widget-tweaks")``
  (`docs <https://docs.python.org/3/library/importlib.metadata.html#distribution-versions>`__).
```